docker-compose.yml

```diff
@@ -1,16 +0,0 @@
-dev:
-  build: .
-  links:
-  - db
-  volumes:
-  - .:/src/graphql-sequelize
-  environment:
-    DB_DATABASE: graphql_sequelize_test
-    DB_USER: graphql_sequelize_test
-    DB_PASSWORD: graphql_sequelize_test
-
-db:
-  image: postgres:9.4
-  environment:
-    POSTGRES_USER: graphql_sequelize_test
-    POSTGRES_PASSWORD: graphql_sequelize_test
```

Comment:
Move it back please, docker-compose and Dockerfile should be at root.
It's possible to have both databases in the same file, it's used for local testing aswell.